crawl-ref/docs/options_guide.txt

```diff
@@ -2353,6 +2353,13 @@ tile_web_mouse_control = true
         WebTiles. Regardless of the value of the setting, the minimap will
         respond to mouse control.
 
+tile_web_mobile_input_helper = auto
```

Comment:
Could you clarify why there needs to be an option? I.e. what's the use case for setting this to either true or false?

Reply:
I didn't find a way of detecting if the user is using a virtual keyboard, so is_mobile() looks for a touch screen instead. I hope that will be correct in most cases, but it's far from perfect. Unless we enable the input helper for everybody, I thought users would benefit from being able to override the detection. I guess the most common use case will be devices featuring both a touch screen and a keyboard.

Reply:
Unfortunately there are devices that are touch capable wuthout using a virtual keyboard, e.g. more and more Windows tablets.
And, equally as unfortunately, I don't know of a better way of detecting "mobile", as neither touch capability nor screen size are indicative any longer.

Reply:
While the current criteria is likely to render many false positives, it is not a big issue IMHO. The input helper is semi-transparent and covers only a small part of the map. We can make it even more discrete if needed. A false negative is a bigger problem because the game becomes unplayable, but I can't imagine a situation where that should happen.

Reply:
Sounds good for now. We may want to eventually figure out some custom detection for some of the more popular windows devices if possible.